src/app/public/modules/definition-list/definition-list.component.ts

```diff
@@ -15,12 +18,20 @@ import { SkyDefinitionListService } from './definition-list.service';
 })
 export class SkyDefinitionListComponent {
 
-  // TODO: figure out how to manage state here.
+/**
+ * Specifies the width of the label portion of the definition list.
+ * @default 90px
+ */
   @Input()
   public set labelWidth(value: string) {
     this.service.labelWidth.next(value);
   }
 
+/**
+ * Specifies the text to display in the value portion of the definition
```

Comment:
Reviewing this, the wording here struck me as awkward and a little difficult to follow. What do you think of changing this to `Specifies a default value to display when no value is provided for a label-value pair.`?  Do you think that makes things any clearer?

Reply:
Updated the description